src/modules/sign/state/AuthContext.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useReducer } from "react";
+
+const LogInContext = React.createContext();
+
+const SET_TOKEN = "SET_TOKEN";
+const INJECT_USER = "INJECT_USER";
+
+const initialState = {
+  token: "",
+  user: "",
+};
+
+const reducer = (state, { type, payload }) => {
+  switch (type) {
+    case SET_TOKEN:
+      return { ...state, token: payload };
+
+    case INJECT_USER:
```

Comment:
Nitpicking here, I think `SET_USER` might be a better alternative here.

Reply:
LOL, Will change it to SET_USER, boss. Thanks.